roles/dsm_opt/defaults/main.yml

```diff
@@ -0,0 +1,18 @@
+servername: ""  # Mandatory
+nodename: ""
+password_access: ""
+password: ""
+domain: []
+include_patterns: []
+exclude_patterns: []
+managedservices: ""
+schedlogname: ""
+errorlogname: ""
+compression: ""
+resourceutilization: ""
+tcpbuffsize: ""
+tcpwindowsize: ""
+httpport: ""
+txnbytelimit: ""
+optfile_path: "/opt/tivoli/tsm/client/ba/bin/dsm.opt"
```

Comment:
variables should really be related to the role.

e.g. `dsm_opt_path` and `dsm_opt_state`

Reply:
I have refactored the variable names, thanks for pointing out that..